src/app.js

```diff
@@ -105,15 +109,33 @@ app.use((_req, res, _next) => {
       scriptSrc: [
         "'self'",
         // Support GA4 per https://developers.google.com/tag-platform/tag-manager/web/csp
-        'https://*.googletagmanager.com'
+        'https://*.googletagmanager.com',
+        'https://*.google-analytics.com',
+        // Support OptinMonster Campaign Embed Code:
+        // https://optinmonster.com/docs/how-to-manually-add-an-after-post-or-inline-optin/#Campaign_Embed_Code
+        'https://*.omappapi.com'
       ],
       imgSrc,
       connectSrc: [
         "'self'",
         // Support GA4 per https://developers.google.com/tag-platform/tag-manager/web/csp
         'https://*.google-analytics.com',
-        'https://*.analytics.google.com',
-        'https://*.googletagmanager.com'
+        'https://*.googletagmanager.com',
+        // Support OptinMonster Campaign Embed Code:
+        // https://optinmonster.com/docs/how-to-manually-add-an-after-post-or-inline-optin/#Campaign_Embed_Code
+        'https://*.omappapi.com'
+      ],
+      styleSrc: [
+        "'self'",
+        "'unsafe-inline'",
+        // Support OptinMonster Campaign Embed Code:
+        // https://optinmonster.com/docs/how-to-manually-add-an-after-post-or-inline-optin/#Campaign_Embed_Code
+        'https://*.omappapi.com',
+        'http://fonts.googleapis.com'
+      ],
+      fontSrc: [
+        "'self'",
+        'http://fonts.gstatic.com'
```

Comment:
Potentially blocking: What’s really not optimal is that OptionMonster is loading font resources over `http` instead of `https`.

Reply:
Hm, yeah this is going to lead to mixed-content warnings I think (if it loads at all) - do any other sites at Mozilla use this services? I'd like to see what kind of CSP they came up with.

Reply:
Yeah, I feel this might be a blocker. In addition to the warnings, I'd like to avoid the ability for ISPs, VPNs, open networks, etc. to MITM these, even if they're just fonts. And I'm not a big fan of adding more fonts to the page load size just for this one component (whatever it is - I don't have the context here).

But perhaps more importantly, needing this makes me suspicious of the development practices of OptinMonster: unnecessarily adding more third-party connections, and over an unencrypted connection at that, does not bode well...

Reply:
Quick update on the unencrypted connection: After checking locally again it seems like — presumably after disabling the OptinMonster logo on the banner — there is no attempt to load resources from http://fonts.gstatic.com anymore. Even though the banner itself does not have any custom fonts used CSS is still being loaded from https://fonts.googleapis.com/css?family=Open+Sans, though.